src/style.css

```
@@ -0,0 +1,123 @@
+
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: 300;
+  padding-top: 20px ;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+input {
+  padding: 0;
+}
+
+.form-big {
+  margin-top: 0;
+  margin-bottom: 20px;
+}
+
+.form-small {
+  margin-top: 0;
+}
+
+.search {
+  width: 100%;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search--big {
+  line-height: 22px;
+  padding-top: 24px;
+  padding-left: 62px;
+  padding-bottom: 22px;
+}
+
+.search::placeholder {
+  font-family: "Avenir", sans-serif;
```

move `font-family` to the `html` tag selector and let inputs inherit it  
```suggestion
  font-family: inherit;
```